package.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "gruposbf-favorites",
+    "version": "0.0.0",
+    "description": "",
+    "scripts": {
+        "build": "npm run clean && tsc --build",
+        "clean": "rm -rf ./_dist/ && mkdir _dist",
+        "start": "npm run build && node ./_dist/program.js",
+        "dev": "npm run build && nodemon ./_dist/program.js",
+        "debug": "set DEBUG=* && npm run dev",
+        "lint": "eslint . --ext .ts",
+        "lint-and-fix": "eslint . --ext .ts --fix"
```

:nail_care: 

isso costuma ser meio que uma convenção para projetos front

```suggestion
        "lint:fix": "eslint . --ext .ts --fix"
```